flake.nix

```diff
@@ -239,9 +238,9 @@
           mkShell {
             nativeBuildInputs = [
               (haskell.lib.justStaticExecutables
-                # (ghcVersion pkgs).fourmolu
-                (import nixpkgs2305 { inherit system; }).haskellPackages.fourmolu_0_12_0_0
```

Comment:
the import of `nixpkgs2305` should be removed entirely now that we no longer need the old versions